frontend/public/branding.ts

```diff
@@ -0,0 +1,11 @@
+const getProductName = () => {
+  switch ((window as any).SERVER_FLAGS.branding) {
+    case 'ocp':
+      return 'OpenShift Container Platform';
```

Comment:
Would change this to enum and use in the [masthead](https://github.com/openshift/console/blob/master/frontend/public/components/masthead.tsx#L101-L117) as well

Reply:
Difficult to change to an enum since it's a value from a config map, set in `SERVER_FLAGS` in JS using a Go template. I think leaving it as a string is simpler and less error prone in this case.

We could add additional validation to the value at startup.

Reply:
sorry for the confusion, I meant to create a enum for the returned values

Reply:
I feel like it doesn't buy much because you still need the switch in masthead, and then you have two values representing the same thing in slightly different ways.

Reply:
ok, I see your point :)